components/QuestChain/QuestChainV1Page.tsx

```diff
@@ -235,6 +254,35 @@ export const QuestChainV1Page: React.FC<QuestChainV1PageProps> = ({
     return memberRoles;
   }, [questChain]);
 
+  const [statusesPoH, setStatusesPoH] = useState<{
+    [addr: string]: boolean;
+  }>({});
+
+  const client = createClient({
+    url: PoHAPI,
+  });
+
+  const getPOH = async () => {
+    const statuses: { [addr: string]: boolean } = {};
+
+    Object.keys(members).forEach(async address => {
```

Comment:
Could we query all members in a single api call instead of looping through them ?

Reply:
done